packages/preset-rules/src/rules.ts

```diff
@@ -15,6 +15,25 @@ export type Priority = (typeof priorities)[number];
 export const defaultPriority: Priority = 'P3';
 export const defaultWcagVersion: WcagVersion = '2.1';
 
+export const disabledRules = [
```

Comment:
Any reason to move these from jest to preset-rule package?

Reply:
There was a Todo to migrate it to the preset-rule package, and I also needed the adaptA11yConfigIncompleteResults function in the browser-lib package. If I hadn't moved it, it would have introduced a dependency on Jest within browser-lib.